xapi/acrossx/.htaccess

```diff
@@ -9,6 +9,8 @@ RewriteCond %{HTTP_ACCEPT} text/html [OR]
 RewriteCond %{HTTP_ACCEPT} application/xhtml\+xml [OR]
 RewriteCond %{HTTP_USER_AGENT} ^Mozilla/.*
 RewriteRule ^([a-z-]+)/?$ http://xapi.vocab.pub/datasets/acrossx/$1 [R=303]
+RewriteRule ^([a-z-]+)/(zh-cn)$ http://xapi.vocab.pub/datasets/acrossx/$1/zh-cn [R=303]
+RewriteRule ^([a-z-]+)/(zh-tw)$ http://xapi.vocab.pub/datasets/acrossx/$1/zh-tw [R=303]
```

Comment:
You could use one generic pattern for these and future language urls:
`RewriteRule ^([a-z-]+)/([a-z-]+)$ http://xapi.vocab.pub/datasets/acrossx/$1/$2 [R=303]`